test/test_metapackage.py

```diff
@@ -62,11 +62,7 @@ def assert_warning(warnreg):
 
 
 def _validate_metapackage(path, package):
-    try:
-        validate_metapackage(path, package)
-    except Exception:
-        # print('on package ' + package.name, file=sys.stderr)
-        raise
+    validate_metapackage(path, package)
```

Comment:
With the meat of this wrapper function removed does it make sense to remove it entirely and call validate_metapackage directly in the tests?

Reply:
Yeah, that makes a lot of sense. Updated with 9eab131c6193024ddb48750d384c3b96b47f60e6, thanks!